cmd/ci-secret-generator/main_test.go

```diff
@@ -228,3 +229,47 @@ func TestValidateContexts(t *testing.T) {
 		})
 	}
 }
+func TestExecuteCommand(t *testing.T) {
+	testCases := []struct {
+		name          string
+		cmd           string
+		expected      []byte
+		expectedError error
+	}{
+		{
+			name:     "basic case",
+			cmd:      "echo basic case",
+			expected: []byte("basic case\n"),
+		},
+		{
+			name:          "error on no output",
+			cmd:           "true",
+			expectedError: fmt.Errorf("command \"true\" returned no output"),
+		},
+		{
+			name:          "error on cmd failure",
+			cmd:           "false",
+			expectedError: fmt.Errorf(" : exit status 1"),
```

Comment:
This message is strange, but I suppose this just tests the current behavior.

Reply:
Yes, it is.